src/app/loans/glim-account/create-glim-account/create-glim-account.component.ts

```
@@ -265,57 +254,10 @@ export class CreateGlimAccountComponent {
-  }
-
-  notify(body: any, context?: { [k: string]: unknown }) {
-    const parts: string[] = [String(body?.defaultUserMessage ?? '')];
-    if (Array.isArray(body?.errors)) {
-      for (const e of body.errors) parts.push(String(e?.developerMessage ?? ''));
-    }
-    if (context) parts.push(`Context: ${JSON.stringify(context)}`);
-    console.error(parts.join(' ').trim());
+  notify(body: any, data: any) {
+    let message = body.defaultUserMessage + ' ';
+    body.errors?.forEach((error: any) => (message += error.developerMessage + ' '));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)